src/main/java/com/openlattice/organizations/HazelcastOrganizationService.java

```diff
@@ -346,20 +324,13 @@ public void removeMembers( UUID organizationId, Set<Principal> members ) {
                         .filter( m -> m.getType().equals( PrincipalType.USER ) )
                         .map( securePrincipalsManager::lookup ) );
         membersOf.submitToKey( organizationId, new OrganizationMemberRemover( members ) );
```

Comment:
Unrelated to this PR-- I think we need execute on key here... submitToKey returns a future and means the call can return before the actual removal is complete.